test/toTex_test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+
+const parse = require('../docs/bundle.js');
+const toTex = require('../lib/toTex.js');
```

Comment:
I'm changing toTex_test.js, but the changes don't show when I run npm test. I am getting the same error even when I remove the test case.